Rakefile

```diff
@@ -1,8 +1,13 @@
 # frozen_string_literal: true
 require 'rubocop/rake_task'
+require 'rake/testtask'
 RuboCop::RakeTask.new
 
 require 'scraper_test'
 ScraperTest::RakeTask.new.install_tasks
 
+Rake::TestTask.new do |t|
```

Comment:
This is going to conflict with `ScraperTest::RakeTask` above, because they both create a task called `test`. 

This makes me think that perhaps `scraper_test`'s rake task should be called `scraper_test` rather than `test`, to make things a bit clearer.

In the meantime you can customize the name of the tasks by passing the task name to the constructor and then changing the default task:

``` ruby
ScraperTest::RakeTask.new.install_tasks(:scraper_test)

task default: %i(rubocop test scraper_test)
```

Does that make sense?